src/config/main.ejs

```diff
@@ -252,6 +347,38 @@ module.exports = {
         browserName: 'firefox'
       }
     },
+    <% } %><% if (answers.mobileRemote) { %>
+    'browserstack.samsung.chrome': {
+      extends: 'browserstack',
+      desiredCapabilities: {
+        browserName: 'chrome',
+        'browserstack.deviceName': "Samsung Galaxy S22"
+      }
+    },
+
+    'browserstack.samsung.local_chrome': {
+      extends: 'browserstack.local',
+      desiredCapabilities: {
+        browserName: 'chrome',
+        'browserstack.deviceName': "Samsung Galaxy S22"
+      }
+    },
+
+    'browserstack.ios.safari': {
+      extends: 'browserstack',
+      desiredCapabilities: {
+        browserName: 'safari',
+        'browserstack.deviceName': "iPhone 13"
+      }
+    },
+
+    'browserstack.samsung.local_chrome': {
+      extends: 'browserstack.local',
+      desiredCapabilities: {
+        browserName: 'safari',
+        'browserstack.deviceName': "iPhone 13"
+      }
+    },
```

Comment:
Should we add one for firefox also? And have you checked these configs are working?

Also, we should only add the browsers present in `answers.remoteBrowsers` here as well, as this is also for remote.

Reply:
Configs are remaining to be checked for correctness.

Reply:
Plus, we only have remote mobile browser configs for BrowserStack, we should also have it for SauceLabs (and for "other" cloud??).

Reply:
As for BStack, we can leave it like now for now and see if people want that added later